src/main/java/uk/nhs/hee/tis/trainee/sync/facade/PlacementEnricherFacade.java

```diff
@@ -178,11 +180,12 @@ private boolean enrich(Placement placement, Site site) {
 
     String siteName = getSiteName(site);
     String siteLocation = getSiteLocation(site);
+    String siteKnownAs = getSiteKnownAs(site);
 
-    if (siteName != null || siteLocation != null) {
+    if (siteName != null || siteLocation != null || siteKnownAs != null) {
```

Comment:
I was wondering if we'd want to populate a placement if the site had no name or location, but did have a 'known as'. I guess we would - but also looking at the data, there are no sites in this condition, so in a sense its a problem we can't tackle 👍

Reply:
Yeah.. it depends do we want to populate site data if the siteName and siteLocation both null. Have removed the  `siteKnownAs != null` condition